catalog/harvest/catalog-harvest-common/src/main/java/org/codice/ddf/catalog/harvest/common/FileSystemPersistenceProvider.java

```diff
@@ -204,7 +209,9 @@ public void clear() {
     File[] files = new File(getMapStorePath()).listFiles(getFilenameFilter());
     if (files != null) {
       for (File file : files) {
-        if (!file.delete()) {
+        try {
+          Files.delete(file.toPath());
+        } catch (IOException e) {
           LOGGER.debug("File was unable to be deleted: {}", file.getAbsolutePath());
```

Comment:
❓Is this going to cause a problem upon restart? Should we log something higher than `debug`?